source/include/FreeRTOSIPConfigDefaults.h

```diff
@@ -3064,7 +3067,7 @@
 
 #ifndef FreeRTOS_debug_printf
```

Comment:
When I wrote [PR #1051](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/1051), I didn't know you had planned to do the change as well.

My proposal would be like this:
~~~
#ifndef FreeRTOS_debug_printf
    #if ( ipconfigHAS_DEBUG_PRINTF == 1 ) && defined( configPRINTF )
        #define FreeRTOS_debug_printf( MSG )    do { configPRINTF( MSG ); } while( ipFALSE_BOOL )
    #else
        #define FreeRTOS_debug_printf( MSG )    do {} while( ipFALSE_BOOL )
    #endif
#endif

#ifndef FreeRTOS_printf
    #if ( ipconfigHAS_PRINTF == 1 ) && defined( configPRINTF )
        #define FreeRTOS_printf( MSG )    do { configPRINTF( MSG ); } while( ipFALSE_BOOL )
    #else
        #define FreeRTOS_printf( MSG )    do {} while( ipFALSE_BOOL )
    #endif
#endif
~~~

Reply:
Let me merge 1051 PR changes in this PR.